src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task{
+    protected String time;
+    protected String desc;
+
+    public Deadline(String description){
```

Comment:
Minor issue but I think a space should be added between ) and {
So it looks like this: "public Deadline(String description) {"
This issue also appears in the rest of the code too.